Cargo.toml

```diff
@@ -21,9 +21,41 @@ clippy.missing_const_for_fn = "deny"
 clippy.too_long_first_doc_paragraph = "warn"
 
 [workspace.dependencies]
-
-[profile.test.package.proptest]
-opt-level = 3
+async-trait = "0.1.83"
+bitcoin = { version = "0.32.5", features = [ # keep in sync with secp256k1
+  "serde",
+] }
+futures = "0.3.31"
+hex = "0.4.3"
+libp2p = { version = "0.55.0", features = [
+  "noise",
+  "gossipsub",
+  "tcp",
+  "request-response",
+  "tokio",
+  "secp256k1",
+  "macros",
+  "ping",
+  "yamux",
+  "identify",
+] }
+libp2p-identity = { version = "0.2.10", default-features = false, features = [
+  "secp256k1",
+  "peerid",
+  "rand",
+] }
```

Comment:
If you are importing `libp2p` above with the `"identity"` feature then you don't need this dep you can use `use libp2p::identity`. Also has the risk of getting out of sync versioning with `libp2p`. So either a good argument to keep it or remove it please?